.github/CONTRIBUTING.md

```diff
@@ -21,7 +21,8 @@ Additionally, if you are interesting in contributing to the codebase, submit a p
 3. Make sure the unit tests pass before raising a PR. For all the unit tests to pass, you typically need to setup a premium proxy service such as `ScraperAPI` or `Luminati` (`Bright Data`). If you do not have an account, you may try to use `FreeProxy`. Without a proxy, 6 out of 17 test cases will be skipped.
 4. Check that the documentatation is consistent with the code. Check that the documentation builds successfully.
 5. Submit a PR, with `develop` as your base branch.
-6. After an initial code review by the maintainers, the unit tests will be run with the `ScraperAPI` key stored in the Github repository. Passing all tests cases is necessary before merging your PR.
+6. Run `pre-commit run --all-files` and ensure that all tests pass.
```

Comment:
I think this should be an optional thing that contributors should do. I'll set up a pre-commit bot to the repo so that the formatting will be applied on Github even if the contributors didn't adhere to the style guide.